contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -670,7 +681,8 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
             (uint256 collateralNotional, ) = _calculateChunkRebalanceNotional(leverageInfo, newLeverageRatio, isLever);
 
             sizes[i] = isLever ? _calculateBorrowUnits(collateralNotional, leverageInfo.action) : collateralNotional;
```

Comment:
Let's leave a note here that even though it says it's calculating units this function can also convert notional value as well.